oracle.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""oracle.py.
+This module presents methods of choosing random student
+in a group to make easily for teacher to ask students
+Methods:
```

Comment:
It would be great to show this detailed description to the user. It would also be greater if the methods would provide descriptions themselves.

Reply:
Now detailed description is available for users with -h (help) option